File: AMS/AMS/Entities/Article.cs
using System;
using System.Collections.Generic;

namespace AMS.Entities
{
    public class Article
    {
        public Article()
        {
            RefereeArticles = new HashSet<RefereeArticle>();
        }

        public int Id { get; set; }
        public string Title { get; set; }
        public string FileName { get; set; }
        public string AuthorFirstName { get; set; }
        public string AuthorLastName { get; set; }
        public string AuthorPhone { get; set; }
        public string AuthorMail { get; set; }
        public string AuthorNotes { get; set; }
        public DateTime CreateDate { get; set; }
        public virtual ICollection<RefereeArticle> RefereeArticles { get; set; }
    }
}

File: AMS_UI/src/app/Interfaces/Article.ts
import { Referee } from "./Referee";

export interface ArticleLight {
    Id : number;
    AuthorName: string;
    Title : string;
    State: string;
}

export interface Article {
    Id : number;
    AuthorFirstName : string;
    AuthorLastName : string;
    AuthorPhone : string;
    AuthorMail : string;
    Title : string;
    CreateDate : string;
    FileName : string;
    State : string;
    ArticleComments : ArticleComments[];
    AuthorNotes : string;
}

export interface ArticleComments{
    Referee : Referee;
    Comment : string;
    Score : number;
    Date : string;
}

export interface ReviewModel{
    ArticleId :number;
    RefereeId :number;
    Comment : string;
    Score : number 
}